pkg/webhook/resources/virtualmachine/mutator_test.go

```diff
@@ -127,7 +127,8 @@ func Test_virtualmachine_mutator(t *testing.T) {
 			}
 			err := clientset.Tracker().Add(settingCpy)
 			assert.Nil(t, err, "Mock resource should add into fake controller tracker")
-			mutator := NewMutator(fakeclients.HarvesterSettingCache(clientset.HarvesterhciV1beta1().Settings))
+			mutator := NewMutator(fakeclients.HarvesterSettingCache(clientset.HarvesterhciV1beta1().Settings),
+				fakeclients.NetworkAttachmentDefinitionCache(clientset.K8sCniCncfIoV1().NetworkAttachmentDefinitions))
```

Comment:
It's better to have test cases in `Test_virtualmachine_mutator`

Reply:
It's not easy to add the unit test. Let's add it later.